go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/connectRN/go-hellosign
+
+go 1.21
```

Comment:
is there any reason this can't be 1.23?

Reply:
@jsalehCRN Because apiserv is 1.21 and I didn't feel like upgrading apiserv today.